### PR TITLE
26 implementation of categories list

### PR DESCRIPTION
**Pull request by NadiiaNanivska**

### CODE REVIEW

File: BookUniverseProject/BookUniverse.Application/MediatR/Books/Queries/GetAllBooksByCategory/GetAllBooksByCategoryQuery.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+using MediatR;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookUniverse.Application.MediatR.Books.Queries.GetAllBooksByCategory
+{
+    public record GetAllBooksByCategoryQuery(int category) : IRequest<Result<IEnumerable<BookDto>>>;
```
Comment (KateYatsiuk) [inline BookUniverseProject/BookUniverse.Application/MediatR/Books/Queries/GetAllBooksByCategory/GetAllBooksByCategoryQuery.cs:12]: Not int category, but int categoryId

File: BookUniverseProject/BookUniverseProject/Models/LayoutModel.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿using BookUniverse.Application.DTOs.CategoryDTOs;
+
+namespace BookUniverse.Web.Models
+{
+    public class LayoutModel
```
Comment (KateYatsiuk) [inline BookUniverseProject/BookUniverseProject/Models/LayoutModel.cs:5]: Please change the name of LayoutModel in such a way that it is clear that the action is performed here with categories

**Review by KateYatsiuk — APPROVED** [review]

File: BookUniverseProject/BookUniverseProject/Models/LayoutModel.cs (hunk 2 above)
Comment (NadiiaNanivska) [reply BookUniverseProject/BookUniverseProject/Models/LayoutModel.cs:5]: This LayoutModel is designed to manage not only categories but also folders and other possible menu items. So it should have more generic name, maybe MenuModel

File: BookUniverseProject/BookUniverse.Application/MediatR/Books/Queries/GetAllBooksByCategory/GetAllBooksByCategoryQuery.cs (hunk 1 above)
Comment (NadiiaNanivska) [reply BookUniverseProject/BookUniverse.Application/MediatR/Books/Queries/GetAllBooksByCategory/GetAllBooksByCategoryQuery.cs]: Done

**Pull request merged**